### ISSUE TITLE
that icon!

### ISSUE THREAD

**Issue by samgribley** [wallabag/wallabagger#175]
folks -- I love Wallabag.  I pay for app.wallabag.it.  But that Chrome toolbar icon is **hideous** and somewhat inappropriate.  Can we change it, please?  I am happy to design another based on the kangaroo figure and generate the files if that's acceptable. 

thanks!

**Comment by prcmt** [comment]
I do quite agree 🤭 I've suggested a new icon in #182 that might be better.

**Comment by Rurik19** [comment]
I think we should make an option, smth like "icon theme" to choose icon from several variants

**Comment by Simounet** [comment]
I disagree with the idea to let users changing the icon. We should avoid making code that is not necessary. I like the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle). We don't want to deal with spaghetti code. We should do a vote for our users to choose the right icon and close this topic once and for all (or a long period at least :D).
What do you think about this @nicosomb @j0k3r @tcitworld ?

**Comment by Rurik19** [comment]
Why it is going to be a spaghetti code? icon name will be stored in options and set by api.

**Comment by Simounet** [comment]
You'll add extra code for such a minor functionality. Plus, I think a single icon is important for branding.

**Comment by Rurik19** [comment]
The branding is important. that's ok. and recognizability too.

**Comment by j0k3r** [comment]
We can provide a different icon set if it’s only related to changing color or really minor update.

For example Reeder on iOS:
![image](https://user-images.githubusercontent.com/62333/73747312-7a6e5780-4757-11ea-9e72-68d1976e19ef.jpg)

Meanwhile I think the icon should be more in relation with the real wallabag logo instead that W (my personal though).

**Comment by prcmt** [comment]
> I think the icon should be more in relation with the real wallabag logo instead that W
I'd agree with that. It does look a little out of place.
